packages/next/src/middleware/sitemaps/handleSitemapRequests.ts

```diff
@@ -36,11 +36,15 @@ export interface HandleSitemapRequestsConfig {
   wpUrl: string;
 
   /**
-   * The pathname to the sitemap index file.
-   *
-   * @example /sitemap.xml, /wp-sitemap.xml, /sitemap_index.xml
+   * The sitemap path for server side
    */
   sitemapIndexPath: string;
+
+  /**
+   * The headless frontend URL
+   */
+  frontendUrl: string;
```

Comment:
We may need to create a typescript interface `GetSItemapPropsConfig` that requires `frontendUrl`, and take it out of `HandleSitemapRequestsConfig` as that is not a valid property for the middleware.

Something like:

```ts
export interface GetSitemapPropsConfig extends HandleSitemapRequestsConfig {
  /**
   * The headless frontend URL
   */
  frontendUrl: string;
}
```